plugins/star-rating/tests.js

```diff
@@ -7,55 +7,255 @@ var API_KEY_ADMIN = "";
+    	it('should return 200 and widget list', function(done) {
+    		API_KEY_ADMIN = testUtils.get("API_KEY_ADMIN");
+            APP_ID = testUtils.get("APP_ID");
+            request.get('/o/feedback/widgets?app_id=' + APP_ID + '&api_key=' + API_KEY_ADMIN)
+            .expect(200)
+            .end(function(err, res) {
+            	if (err) return done(err);
+            	var ob = JSON.parse(res.text);
+            	testUtils.set('WIDGET_ID', ob[0]._id);
+            	ob.length.should.above(0);
+            	setTimeout(done, 100);
+            });
+    	});
+    });
+    
+    describe('Get single widget', function() {
+    	it('should return single widget', function(done) {
+    		API_KEY_ADMIN = testUtils.get("API_KEY_ADMIN");
+            APP_ID = testUtils.get("APP_ID");
+            WIDGET_ID = testUtils.get("WIDGET_ID");
+            request.get('/o/feedback/widget?app_id=' + APP_ID + '&api_key=' + API_KEY_ADMIN + '&widget_id=' + WIDGET_ID)
+            .expect(200)
+            .end(function(err, res) {
+            	if (err) return done(err);
+            	setTimeout(done, 100);
+            });
+    	});
+    });
+
+    
+    describe('Get widgets by array parameter', function() {
+    	it('should return widgets array', function(done) {
+    		API_KEY_ADMIN = testUtils.get('API_KEY_ADMIN');
+    		APP_ID = testUtils.get("APP_ID");
+    		WIDGET_ID = testUtils.get('WIDGET_ID');
+    		var array4query = [WIDGET_ID];
+    		request.get('/o/feedback/multiple-widgets-by-id?app_id=' + APP_ID + '&api_key=' + API_KEY_ADMIN + '&widgets=' + JSON.stringify(array4query))
+    		.expect(200)
+    		.end(function(err, res) {
+    			if (err) return done(err);
+    			var ob = JSON.parse(res.text);
+    			ob.length.should.eql(1);
+    			setTimeout(done, 100);
+    		});
+    	});
+    });
+    
+    describe('Editing widget', function() {
+        it('should success', function(done) {
+            API_KEY_ADMIN = testUtils.get("API_KEY_ADMIN");
+            APP_ID = testUtils.get("APP_ID");
+            WIDGET_ID = testUtils.get('WIDGET_ID');
+            request.get('/i/feedback/widgets/edit?api_key=' + API_KEY_ADMIN + '&app_id=' + APP_ID + '&widget_id='+WIDGET_ID+'&popup_header_text=Widget which edited by test&popup_comment_callout=Leave to us a comment&popup_email_callout=Contact me by email&popup_button_callout=Submit Feedback&popup_thanks_message=Thank you for great feedback!&trigger_position=mright&trigger_bg_color=#345678&trigger_font_color=#111222&trigger_button_text=Feedback Trigger Text&target_devices={desktop:true,phone:true,tablet:true}&target_page=selected&target_pages=["/home"]&is_active=false&hide_sticker=false')
+            .expect(200)
+            .end(function(err, res) {
+            	if (err) return done(err);
+            	var ob = JSON.parse(res.text);
+            	ob.should.have.property('result','Success');
+            	setTimeout(done, 100);
+            });
+        });
+    });
+	
```

Comment:
Here it would be great to fetch widgets one more time, to ensure changes were stored in db and saved correctly.